crates/e2e-tests/src/tests.rs

```diff
@@ -73,7 +73,7 @@ async fn test_wallet_api() -> Result<(), Box<dyn std::error::Error>> {
     let tx =
         TransactionRequest::default().with_authorization_list(vec![auth]).with_to(signer.address());
 
-    let tx_hash: B256 = provider.client().request("wallet_sendTransaction", vec![tx]).await?;
+    let tx_hash: B256 = provider.client().request("odyssey_sendTransaction", vec![tx]).await?;
```

Comment:
could you duplicate the test and run one as is and the other one with `odyssey_sendTransaction` . this way we know both endpoints still work (we will remove `wallet_sendTransaction` once we've migrated fully to `odyssey_sendTransaction`)

Reply:
Sure